src/ANSITerminal_win.ml

```diff
@@ -113,11 +113,20 @@ let print ch styles txt =
 let print_string = print stdout
 let prerr_string = print stderr
 
-let printf style = kprintf (print_string style)
-
-let eprintf style = ksprintf (prerr_string style)
-
-let sprintf style = sprintf
+module Printf = struct
+  let printf style = kprintf (print_string style)
+  let eprintf style = ksprintf (prerr_string style)
+  let sprintf _style = sprintf
+  let fprintf _ _ _ = invalid_arg "not implemented"
+end
+
+module Format = struct
```

Comment:
Functions of this module need to be implemented on Windows as well — make it a separated commit if you wish.